manpages/brew.1

```diff
@@ -1,7 +1,7 @@
 .\" generated with Ronn/v0.7.3
 .\" http://github.com/rtomayko/ronn/tree/0.7.3
 .
-.TH "BREW" "1" "October 2021" "Homebrew" "brew"
+.TH "BREW" "1" "November 2021" "Homebrew" "brew"
```

Comment:
Is the `brew brew` workflow not running? Or does it not do anything when only the date changes?

Reply:
Yeah, when it checks for updates it ignores date changes